on-prem-docs/separating-backend-services.md

```diff
@@ -0,0 +1,55 @@
+# Separating Backend Services (minio/postgresql)
+
+The on-prem-builder install.sh script now supports separation of the backend components i.e datastore and minio server onto different nodes.
+You can now have a setup where the postgresql service runs on one node and minio runs on another. Both can be designed to communicate to one another and to a third or more nodes running the front-end services. 
+
+## Pre-requisites
+The bldr.env file for your single on-prem builder node contains most of the information required setup minio and postgresql on it and will be used during the installation process.
+
+Modify your `bldr.env` based on the same config in `bldr.env.sample`
+```bash
+cp bldr.env.sample bldr.env
+```
+
+For the setting up the minio server on a separate node, make sure that `S3_ENABLED` and `ARTIFACTORY_ENABLED` in the bldr.env are set to `false`.
+This must be ensured as minio server can not be used if you are using S3 or Artifactory directly.
+
+For the setting up the datastore on a separate node, make sure that `PG_EXT_ENABLED` in the bldr.env is set to `false`.
+This must be ensured as the Datastore node cannot have externally hosted PostgreSQL(RDS, Azure Database for PostgreSql etc).
+
```

Comment:
I think we can remove lines 14-18 here. As long as the user does not edit what is already in the bldr.env.sample file, these settings will be correct.

Reply:
Agree. I added these checks just to ensure the correct value is set as the fields ARTIFACTORY_ENABLED and PG_EXT_ENABLED are commented out in the bldr.env.sample file. Hence there might be a very small chance that user might have a undesirable value set in their environment that might get picked up.